connection.go

```diff
@@ -44,6 +44,12 @@ const (
 	// LogUnexpectedResultId is logged when response with unknown id were received.
 	// Most probably it is due to request timeout.
 	LogUnexpectedResultId
+	// LogConnected is logged when connection is established or reestablished
+	LogConnected
+	// LogDisconnected is logged when connection is broken
+	LogDisconnected
+	// LogClosed is logged when connection is closed forever
+	LogClosed
```

Comment:
We already have a mechanism for notification of connection/disconnection. Maybe it would be better to add error handling to it?

https://github.com/tarantool/go-tarantool/blob/7897bafc1b6b8a292afb33e57a813791e46bf613/connection.go#L196-L198

https://github.com/tarantool/go-tarantool/blob/7897bafc1b6b8a292afb33e57a813791e46bf613/connection.go#L586-L593

Is there a reason why Opts.Notify cannot be used for your purposes? I see that your log events duplicate Connection.notify().